zone/tribute.cpp

```diff
@@ -375,62 +377,68 @@ void Client::SendGuildTributes() {
 	}
 }
 
-bool ZoneDatabase::LoadTributes() {
-
-	TributeData tributeData;
-	memset(&tributeData.tiers, 0, sizeof(tributeData.tiers));
-	tributeData.tier_count = 0;
+bool ZoneDatabase::LoadTributes()
```

Comment:
Wasn't this code touched in Neckollas Guild PR? Would be good to not make even more work for him to merge at this point. I want to say he already used repositories.

Reply:
He did.